account-management-api/src/main/java/uk/gov/di/accountmanagement/services/AccountDeletionService.java

```diff
@@ -86,10 +87,12 @@ public DeletedAccountIdentifiers removeAccount(
 
         String persistentSessionID = AuditService.UNKNOWN;
         String ipAddress = AuditService.UNKNOWN;
+        Boolean manualDeletion = Boolean.TRUE;
         if (input.isPresent()) {
             ipAddress = PersistentIdHelper.extractPersistentIdFromHeaders(input.get().getHeaders());
             attachLogFieldToLogs(PERSISTENT_SESSION_ID, ipAddress);
             ipAddress = IpAddressHelper.extractIpAddress(input.get());
+            manualDeletion = Boolean.FALSE;
```

Comment:
Is there a reason we are using boxed booleans rather than primitive e.g. `true` and `false`?